### PR TITLE
fixed loading on switching between tabs

### PR DESCRIPTION
**Pull request by NatalyMinziuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/KadraVykhovnykiv/KVTable.tsx [hunk 1]
```diff
@@ -21,14 +21,9 @@ export const KVTable = ({ current, searchData }: props) => {
...
-  const [loading, setLoading] = useState(false);
+  const [isLoading, setLoading] = useState(false);
 
-  const [data, setData] = useState<any[]>([{
-    id: '',
-    user: '',
-    dateOfGranting: '',
-    numberInRegister: '',
-  }]);
+  const [data, setData] = useState<any[]>([]);
```
Comment (danylo-melnychuk) [inline src/pages/KadraVykhovnykiv/KVTable.tsx:26]: Pls, do not use any.

File: src/pages/KadraVykhovnykiv/KVTable.tsx [hunk 2]
```diff
@@ -113,11 +108,11 @@ export const KVTable = ({ current, searchData }: props) => {
 
 
   return (
-    <div>
+    <div style={{textAlign:'center'}}>
```
Comment (ostappppppp) [inline src/pages/KadraVykhovnykiv/KVTable.tsx:111]: Is it possible to do not use inline styles?

Comment (NatalyMinziuk) [reply src/pages/KadraVykhovnykiv/KVTable.tsx]: fixed

**Review by DimaDiD — APPROVED** [review]

File: src/pages/KadraVykhovnykiv/KVTable.tsx (hunk 1 above)
Comment (NatalyMinziuk) [reply src/pages/KadraVykhovnykiv/KVTable.tsx]: fixed

**Review by VSBalanchuk — APPROVED** [review]

**Review by andrii52 — APPROVED** [review]

**Pull request merged**